fitnessllm_dataplatform/task_handler.py

```
@@ -0,0 +1,63 @@
+            InfrastructureNames = DynamicEnum.from_dict(
+                get_secret(environ["INFRASTRUCTURE_SECRET"])[environ["STAGE"]],
+            )
+        except KeyError as e:
+            raise KeyError(f"Missing environment variable: {e}")
+
+        data_source = options['data_source']
+        if data_source == FitnessLLMDataSource.STRAVA.value:
+            strava_api_interface = StravaAPIInterface()
+            try:
+                strava_api_interface.get_all_data(InfrastructureNames)
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)